src/types/globals/ClipboardEvent.js

```diff
@@ -1,6 +1,7 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
```

Comment:
I'm not sure these libdef should have `@flow`, maybe we should make eslint ignore this directory.
But it's not a big deal either so I'm inclined to leave it like you did.